### PR TITLE
Add fluentd args

### PR DESCRIPTION
**Pull request by obeyler**
#### What this PR does / why we need it:
It allows to change the default FLUENTD_ARGS (fluentd--no-supervisor -q) to add by example -vv (fluentd--no-supervisor -q -vv) 
why I need it: I want to be able to run fluentd with full log trace 
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://developercertificate.org) signed
- [X] Chart Version bumped (if the pr is an update to an existing chart)
- [X] Variables are documented in the README.md

### CODE REVIEW

File: charts/fluentd-elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: fluentd-elasticsearch
-version: 2.9.1
+version: 2.9.2
```
Comment (monotek) [inline charts/fluentd-elasticsearch/Chart.yaml]: new feature should raise minor version.

File: charts/fluentd-elasticsearch/values.yaml [hunk 2]
```diff
@@ -49,6 +49,10 @@ elasticsearch:
   buffer_queue_limit: 8
   logstash_prefix: 'logstash'
 
+# If you want to change args of fluentd process
+# by example you can add -vv to launch with trace log
+fluentd_args: "fluentd--no-supervisor -q"
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: Camelcase should be used -> fluentdArgs.
Value should be "--no-supervisor -q".

File: charts/fluentd-elasticsearch/README.md [hunk 3]
```diff
@@ -60,6 +60,7 @@ The following table lists the configurable parameters of the Fluentd elasticsear
 | `elasticsearch.buffer_queue_limit`   | Elasticsearch buffer queue limit                                               | `8`                                    |
 | `elasticsearch.scheme`               | Elasticsearch scheme setting                                                   | `http`                                 |
 | `env`                                | List of env vars that are added to the fluentd pods                            | `{}`                                   |
+| `fluentd_args`                       | Fluentd args                                                                   | `fluentd--no-supervisor -q`            |
```
Comment (monotek) [inline charts/fluentd-elasticsearch/README.md]: value should be: --no-supervisor -q

Comment (obeyler) [reply charts/fluentd-elasticsearch/README.md]: Ok I do it :-)

File: charts/fluentd-elasticsearch/values.yaml (hunk 2 above)
Comment (obeyler) [reply charts/fluentd-elasticsearch/values.yaml]: I do it

**Comment by monotek** [comment]
Please resolve merge conflicts and use camelcase for the variable.

**Pull request merged**